### ISSUE TITLE
broken link

### ISSUE THREAD

**Issue by turkulerc** [stefvanbuuren/fimdbook#13]
section 1.5 refers to www.multiple-imputation.com, but the website cannot be found.

**Comment by stefvanbuuren** [comment]
Yes, I stopped updating it, so it's no longer online. Use wayback to see how it was once.
